ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
Isn't this file already in develop?

Reply:
@CoryMartin-NOAA yes, it it. But this file has been updated for the adpupa prepbufr using "BUFR_py = ["satwind_amv_goes", "adpupa_prepbufr"]"

Reply:
the PR indicates that this is a new file, which is why I'm confused

Reply:
@CoryMartin-NOAA I removed this file. This is already in develop, thanks.